services/slam/builtin/cartographer_int_test.go

```diff
@@ -112,6 +129,9 @@ func TestCartographerIntegration(t *testing.T) {
 	// Don't clear out the directory, since we will re-use the data for the next run
 	closeOutSLAMService(t, "")
 
+	// added sleep to ensure cartographer stops
+	time.Sleep(time.Millisecond * cartoSleepMS)
```

Comment:
why doesnt this sleep exist in `orbslam_int_test.go`

Reply:
As far as I am aware, orbslam was not having an issue where maps/files were getting generated during the next test(which is what prompted me to add the sleeps). If that is an issue that is occurring for orbslam then I can add them there as well.